view.php

```diff
@@ -104,9 +104,9 @@
       . "<a href='$wwsetlink'>" . get_string("iframeNoShow-2", "wwassignment")
       ."</a><p align='center'></iframe></p>\n"
       );
-print("<iframe id='wwPage' src='$wwsetlink' frameborder='1' "
+print("<iframe id='wwPage' src='$wwsetlink' frameborder='0' "
       . "width='".$CFG->wwassignment_iframewidth."' "
-      . "height='".$CFG->wwassignment_iframeheight."'>"
+      . "height='".$CFG->wwassignment_iframeheight."' scrolling='no' >"
```

Comment:
Thanks Michael for the links!
I was working on the wwassignment module... to get it properly I should do an upgrade of the table:
add field introformat and rename description into intro, if you agree with that change the mod_form should be updated to use the intro instead...
You can look at the module of your choice to see those fields (ex mod_label : https://github.com/moodle/moodle/blob/master/mod/label/db/install.xml
)
Do you agree if I do that?